llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -29,6 +29,33 @@ using namespace llvm;
 
 namespace {
 
+static int64_t calculateCSRSpillOffsets(MachineFrameInfo &MFI,
+                                        const TargetFrameLowering *TFI,
+                                        int MinCSFI, int FrameIdx) {
+  int LocalAreaOffset = -TFI->getOffsetOfLocalArea();
```

Comment:
Why is this negated instead of using the raw value? Is this assuming the stack grows down?

Reply:
Yes, here I assume, that for RISC-V stack grows down.